modAionImpl/src/org/aion/zero/impl/sync/TaskImportBlocks.java

```diff
@@ -544,28 +544,30 @@ private ImportResult importBlock(AionBlock b, String displayId, PeerState state)
         if (log.isDebugEnabled()) {
             // printing sync mode only when debug is enabled
             log.debug(
-                    "<import-status: node = {}, sync mode = {}, hash = {}, number = {}, txs = {}, block time = {}, result = {}, time elapsed = {} ms>",
+                    "<import-status: node = {}, sync mode = {}, hash = {}, number = {}, txs = {}, block time = {}, result = {}, time elapsed = {} ms, td = {}>",
                     displayId,
                     (state != null ? state.getMode() : NORMAL),
                     b.getShortHash(),
                     b.getNumber(),
                     b.getTransactionsList().size(),
                     b.getTimestamp(),
                     importResult,
-                    t2 - t1);
+                    t2 - t1,
+                    chain.getTotalDifficulty());
         } else {
             // not printing this message when the state is in fast mode with no parent result
             // a different message will be printed to indicate the storage of blocks
-            if (log.isInfoEnabled()
```

Comment:
this change and the ones below it should be reverted

Reply:
reverted.